themes/dark/templates/displayShow.mako

```diff
@@ -35,7 +35,7 @@ const startVue = () => {
 <%include file="/partials/showheader.mako"/>
 
 <div class="row">
-    <div class="col-md-12 horizontal-scroll">
+    <div class="col-md-12 horizontal-scroll" style="top: 12px">
```

Comment:
Cant use a class for this one?

Reply:
I don't see a reason to make a class for this little thing which is used once...